MediaManager/Plugin.MediaManager.Android/MediaManagerImplementation.cs

```diff
@@ -128,6 +141,27 @@ public void UnbindMediaPlayerService()
             }
         }
 
+        public async Task Play(IMediaFile mediaFile)
+        {
+            switch (mediaFile.Type)
+            {
+                case MediaFileType.AudioUrl:
+                    await Play(mediaFile.Url);
+                    break;
+                case MediaFileType.VideoUrl:
+                    break;
```

Comment:
Why not throw an NotImplemented exception for now, because no implementation exists?